mne/_fiff/_digitization.py

```diff
@@ -225,7 +225,7 @@ def write_dig(fname, pts, coord_frame=None, *, overwrite=False, verbose=None):
         if len(bad_frames) > 0:
             raise ValueError(
                 "Points have coord_frame entries that are incompatible with "
-                "coord_frame=%i: %s." % (coord_frame, str(tuple(bad_frames)))
+                f"coord_frame={coord_frame}: {str(tuple(bad_frames))}."
```

Comment:
Some of these I think could be changed to this, but not 100% necessary
```suggestion
                f"coord_frame={coord_frame}: {tuple(bad_frames)}."
```

Reply:
thanks! added in https://github.com/mne-tools/mne-python/pull/12758/commits/ab42a75bbc705795eba670a061b7530728c2007e